apps/agent/scripts/ganache-cli.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
This file will live on `@aragon/contract-helpers-test`. I include it locally until we publish the new version.